source/stylesheets/site.css.scss

```diff
@@ -263,4 +261,155 @@ footer {
 #map {
   width: 100%;
   height: 300px;
+  margin-bottom: 1rem;
+}
+
+.cover-image {
+  display: block;
+  width: 100%;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+}
+.gallery {
+  padding: 1rem;
+  display: grid;
+  grid-gap: 1rem;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
```

Comment:
@EnigmaEdivad Estrai il valore `200px` in una variabile, in modo che sia facilmente configurabile nella sezione delle variabili.